### PR TITLE
CCDB: Update disclaimer content and modal link

### PR DESCRIPTION
**Pull request by anselmbradford**
## Removals

- Remove the three paragraphs of disclaimer text

## Changes

- Change the text of the link that opens the modal to "Things to know before you use this database"
- Update the text of the modal (see the additional context section below for the full text)

## Testing

- Pull branch and click "Things to know before you use this database" link.
 
## Screenshots
Screen Shot 2022-02-15 at 2 00 27 PM

Screen Shot 2022-02-15 at 2 00 34 PM

### CODE REVIEW

File: src/components/Dialogs/MoreAbout.jsx [hunk 1]
```diff
@@ -2,46 +2,58 @@ import React from 'react'
...
                rel="noopener noreferrer">
-              publication criteria.
-            </a>
-            &nbsp;Data is generally updated daily.
-          </p>
-          <p className="body-copy">
-            Company specific information should be considered in the context of
-            that company's size and/or market share.
+              publication criteria
+            </a>.
```
Comment (niqjohnson) [inline src/components/Dialogs/MoreAbout.jsx:56]: @anselmbradford, sorry for the churn, but this content has been rearranged slightly and some of the links have been updated. Here's the new content:

> ## Things you should know before you use this database
> 
> This database only includes complaints the CFPB sent to companies and complaints are only published after the company responds, confirming a commercial relationship or after 15 days, whichever comes first. This database does not include complaints referred to other regulators, such as complaints about depository institutions with less than $10 billion in assets. 
> 
> This database is not a statistical sample of consumers’ experiences in the marketplace. Complaints are not necessarily representative of all consumers’ experiences with a financial product or company. Company-specific information should be considered in the context of that company’s size and/or market share.
> 
> This database only includes a consumer’s description of what happened in their own words if the consumer agrees to share it and after we take steps to [remove personal information](https://files.consumerfinance.gov/f/documents/201503_cfpb_Narrative-Scrubbing-Standard.pdf). One consumer’s experience is not necessarily representative of all consumers’ experiences and narratives are not verified before publication.
> 
> This database generally updates daily and doesn’t include complaints that don’t meet our [publication criteria](https://files.consumerfinance.gov/f/documents/201503_cfpb_disclosure-of-consumer-complaint-narrative-data.pdf).

**Review by niqjohnson — CHANGES_REQUESTED** [review]
The modal content has been updated a little—sorry for the churn!

File: src/components/Dialogs/MoreAbout.jsx (hunk 1 above)
Comment (anselmbradford) [reply src/components/Dialogs/MoreAbout.jsx]: Updated in https://github.com/cfpb/ccdb5-ui/pull/387/commits/d891910a16d6dd21dfa5b013adde7a1bc4dd820e

Screen Shot 2022-02-15 at 4 07 38 PM

**Review by niqjohnson — COMMENTED** [review]
This looks great to me. I caught one thing that's unrelated to the content change specifically but that we might want to address here. Looks like paragraphs in the modal have a little extra margin applied to them somewhere:

```css
.modal-body .body p {
    margin-bottom: 30px;
}
```

That looks extra weird now that there's more content in the modal. Can we get rid of that rule? Our regular 15px margin between paragraphs looks just fine in the modal.

**Comment by anselmbradford** [comment]
@niqjohnson Cool, removing stuff is good! Updated in https://github.com/cfpb/ccdb5-ui/pull/387/commits/0f13345e1024efc0e5706d0326688a12162a917e
Screen Shot 2022-02-16 at 11 36 59 AM

**Review by niqjohnson — APPROVED** [review]
Looks great!

**Pull request merged**